src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.scss';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
```

Comment:
about CSS, I think we can follow our guideline on the [compass](https://compass.nimblehq.co/development/code-conventions/css/)

Reply:
Updated!